network-api/networkapi/wagtailpages/templates/wagtailpages/pages/youtube-regrets-2022/fragments/hero_carousel.html

```diff
@@ -0,0 +1,71 @@
+      <p class="heading-4 text-white tw-mt-2">{% trans "People said:" %}</p>
+
+      {# Controls #}
+      <div class="carousel-hero__controls mb-4 order-4 order-lg-3">
+        <div class="carousel-hero__controls-inner carousel__count-inner--hero">
+          <div class="d-flex align-items-center justify-content-end">
+            <div class="carousel-hero__count">
+              <div class="carousel__count-inner text-white d-flex align-items-baseline justify-content-end"></div>
+            </div>
+            <button class="carousel-hero__button carousel__button--prev button mr-3" aria-label="{% trans "Previous slide" %}">
+              <svg aria-hidden="true" width="41" height="41" fill="none" xmlns="http://www.w3.org/2000/svg"><path class="circle" d="M20.48 39.736c10.586 0 19.167-8.581 19.167-19.167 0-10.585-8.58-19.166-19.166-19.166S1.314 9.983 1.314 20.569s8.581 19.167 19.167 19.167z" stroke="#000" stroke-linecap="round" stroke-linejoin="round"/><path class="arrow" d="M20.48 12.903l-7.666 7.666 7.667 7.667M28.147 20.57H12.814" stroke="#000" stroke-linecap="round" stroke-linejoin="round"/></svg>
+            </button>
+            <button class="carousel-hero__button carousel__button--next button" aria-label="{% trans "Next slide" %}" >
+              <svg aria-hidden="true" width="41" height="41" fill="none" xmlns="http://www.w3.org/2000/svg"><path class="circle" d="M20.48 39.736c10.586 0 19.167-8.581 19.167-19.167 0-10.585-8.58-19.166-19.166-19.166S1.314 9.983 1.314 20.569s8.581 19.167 19.167 19.167z" stroke="#000" stroke-linecap="round" stroke-linejoin="round"/><path class="arrow" d="M20.48 28.236l7.667-7.667-7.666-7.666M12.814 20.57h15.333" stroke="#000" stroke-linecap="round" stroke-linejoin="round"/></svg>
+            </button>
+          </div>
+        </div>
+      </div>
+
+      <div class="carousel-hero__slides swiper-wrapper order-3 order-lg-4" aria-live="polite">
+        {# Slide 1 #}
+        <div class="carousel-hero__item swiper-slide">
+          <div class="carousel-hero__video">
+            <img class="tw-w-full large:w-auto large:tw-float-right mb-2" src="{% static "_images/youtube-regrets/hero-carousel/2022/carousel-1.jpg" %}">
+          </div>
+          <div class="carousel-hero__item-content">
+            <p class="carousel-hero__item-text">
+              {% blocktrans trimmed %}
+                <span class="tw-text-youtube-regrets-red">I avoided clicking a video</span> I would like to watch, only because I was worried that doing so would lead me to get politically extreme recommendations.
+              {% endblocktrans %}
+            </p>
+          </div>
+        </div>
```

Comment:
**Suggestion** This feels like a good place to break out a component to avoid the duplication.